packages/edit-site/src/components/header/fullscreen-mode-close/index.js

```diff
@@ -21,6 +22,10 @@ function FullscreenModeClose() {
 		return null;
 	}
 
+	if ( applyFilters( 'siteEditor.closeButton.remove', false ) ) {
```

Comment:
I think the convention is to use the full function name. 

```js
siteEditor.fullscreenModeClose.remove
```

Reply:
I removed the filter in the latest refactor so this is no longer relevant.